### DISCUSSION TITLE
FIleProcessor notes

### DISCUSSION THREAD

**Discussion by sakkammadam** [sakkammadam/scratchStuff#3] [General]
@hbaird7877 / Abraham - 
 Please find the changes for FIleProcessor readDirectory method setup within the FileProcessor.cpp file.
 I haven't started on the outputs just yet.
 I would like us to start on the Mapper functionality before proceeding.
 Hope to meet you tomorrow.

Thanks,
Sooraj

**Comment by hbaird7877** [comment]
Sooraj -
Your plan makes sense.
I am conducting research regarding how to install and configure Boost libraries and also which functions from Boost 
have potential utility for our MapReduce phase 1 project.

Thanks, 
Hal
